opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -880,25 +886,31 @@ namespace Opm {
     template<typename TypeTag>
     SimulatorReport
     BlackoilWellModel<TypeTag>::
-    solveWellEq(const std::vector<Scalar>& B_avg, const double dt, Opm::DeferredLogger& deferred_logger)
+    solveWellEq(const double dt, Opm::DeferredLogger& deferred_logger)
     {
         WellState well_state0 = well_state_;
 
+        if( wellsActive() )
+        {
```

Comment:
Did you see some values are not initialized under some circumstances?

Reply:
I did. But now the code seems to work without. I will remove the change.